docs/user.md

```diff
@@ -0,0 +1,54 @@
+# User
+
+#### Three sample users in current user database:
+
+* email: 123@123.com password:123
+* email: michael@google.com  password:123
+* email: admin@backtesting.com password: admin
+
+(feel free to create more through register page)
+
+
+####To enforce a user to access web page after login 
+
+Add login_require decorator under the endpoint route
+```sh
+@login_required
+```
+(eg. /home /account)
+
+####To get a current user's info
+use object: current_user
+
+for example, after user Michael log in
+```sh
+print(current_user)
```

Comment:
how can I get `current-user` object?

Reply:
in line12 current_user is imported from flask_login which will handle the login session, so you can use it directly in app.py